jwst/guider_cds/guider_cds_step.py

```diff
@@ -21,7 +23,29 @@ class GuiderCdsStep (Step):
 
     def process(self, input):
         with datamodels.GuiderRawModel(input) as input_model:
-            out_model = guider_cds.guider_cds(input_model)
+
+            # Get the gain reference file
+            gain_model = None   # will overwrite if file can be retrieved
+            try:
+                gain_filename = self.get_reference_file(input_model, 'gain')
+                gain_model = datamodels.GainModel(gain_filename)
+                self.log.info('Using GAIN reference file: %s', gain_filename)
+            except CrdsLookupError as err:
+                msg = f'{err} Unable to retrieve GAIN ref file.'
```

Comment:
Is there any benefit in including the `err` message here and then adding another message, especially using f-strings notation in a log message? If there's no particular benefit `err` should be omitted.